### ISSUE TITLE
Should electron-squirrel-startup be included?

### ISSUE THREAD

**Issue by zeke** [electron/update-electron-app#20]
Hi @imlucas 👋 

I just came across [electron-squirrel-startup](https://github.com/mongodb-js/electron-squirrel-startup). Do you think it would make sense for this module (update-electron-app) to require and invoke it?

cc @MarshallOfSound

**Comment by imlucas** [comment]
@zeke makes sense to me. It’s overdue for some tlc... @durran has some upcoming auto updates work as well, so either of you do lmk if there are any code warts I can take care of to make this easier.

**Comment by zeke** [comment]
Thanks for the reply 👍 

> overdue for some tlc

Can you elaborate?

**Comment by MarshallOfSound** [comment]
Thanks for the question. The project settled on recommending electron-squirrel-startup separately and the README FAQ already covers this, so we are closing this 2018 question as stale. Please open a fresh issue if you still have questions on a current version.
